### PR TITLE
feat(ci): publish GitHub pages site with PR information (no fork)

### PR DESCRIPTION
**Pull request by wdconinc**
### Briefly, what does this PR introduce?
See #892. This is a mirror of that PR but on a branch inside github.com/eic/EICrecon.

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -1,5 +1 @@
-# EICrecon
-
-**JANA based reconstruction for the EPIC detector**
-
-<img src="demo.png" style="max-height: 300px"/>
+Replaced
```
Comment (veprbl) [inline docs/README.md:1]: Please revert, this was merged accidentally

File: .github/workflows/linux-eic-shell.yml [hunk 2]
```diff
@@ -514,27 +511,110 @@ jobs:
...
+        with:
+          include.*.pr: ${{ toJSON(fromJSON(steps.query.outputs.data).*.number) }}
+
+  get-docs-from-open-prs:
+    runs-on: ubuntu-latest
+    needs:
+      - build-docs
+      - list-open-prs
+    strategy:
+      matrix: ${{ fromJson(needs.list-open-prs.outputs.json) }}
```
Comment (veprbl) [inline .github/workflows/linux-eic-shell.yml:547]:
```suggestion
      matrix: ${{ fromJSON(needs.list-open-prs.outputs.json) }}
```
This is official spelling https://docs.github.com/en/actions/learn-github-actions/expressions#fromjson

File: docs/README.md (hunk 1 above)
Comment (veprbl) [reply docs/README.md:1]:
```suggestion
# EICrecon

**JANA based reconstruction for the EPIC detector**

<img src="demo.png" style="max-height: 300px"/>
```

Comment (wdconinc) [reply docs/README.md]: Reverted.

**Review by veprbl — APPROVED** [review]
https://eicrecon.epic-eic.org/pr/923/#/ works.

**Comment by wdconinc** [comment]
Release the capybara!

**Pull request merged**